OKTA-567446 Add settings.aaguidGroups to the Authenticator methods endpoint (#3938)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/reference/api/authenticators-admin/index.md

| `status` | String (Enum) | The status of the authenticator method. Possible values: `ACTIVE` or `INACTIVE` | All Authenticator Methods |
| `type` | String (Enum) | The type of authenticator method. Possible values: `cert`, `duo`, `email`, `idp`, `otp`, `password`, `push`, `security_question`, `signed_nonce`, `sms`, `totp`, `voice`, or `webauthn`  |  All Authenticator Methods |

### Authenticator Method Settings object

#### Authenticator Method Settings properties

> **Note:** The `aaguidGroups` property supports the [Early Access](/docs/reference/releases-at-okta/#early-access-ea) (Self-Service) Allow List for FIDO2 (WebAuthn) Authenticators feature. Enable the feature for your org from the **Settings** > **Features** page in the Admin Console.

| Property | Type | Description  | Applies to Authenticator Method type |
| -------- | ---- | ------------ | ------------------------------------ |
| `aaguidGroups` <ApiLifecycle access="ea" /> | Array of [AAGUID Group](#aaguid-group-object) | The FIDO2 AAGUID groups available to the WebAuthn authenticator. | `webauthn` |

> | `acceptableAdjacentIntervals` | Integer | Number of acceptable adjacent intervals, also known as the clock drift interval. This setting allows you to build in tolerance for any time difference between the token and the server. For example, with a `timeIntervalInSeconds` of 60 seconds and an `acceptableAdjacentIntervals` value of 5, the result of 60 X 5 is 300, which means that Okta accepts passcodes within 300 seconds before or after the end user enters their code. Possible values: 0&ndash;10 | `otp` |

Reviewer: ```suggestion
| `acceptableAdjacentIntervals` | Integer | The number of acceptable adjacent intervals, also known as the clock drift interval. This setting allows you to account for time differences between tokens and servers. For example, with a `timeIntervalInSeconds` of 60 seconds and an `acceptableAdjacentIntervals` value of 5, Okta would accept passcodes within 300 seconds (60 * 5) before or after the end user enters their code. Possible values: 0&ndash;10 | `otp` |
```

Reviewer: MS style guide calls for an `×` (or sometimes an `*`) symbol instead of an X.

https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/m/multiplication-sign

Author: Thanks! I updated the description to use MS style * instead of X. I kept the "build in tolerance" line to be consistent with Product doc description.
